util/config/config.go

```diff
@@ -78,11 +78,14 @@ const (
 )
 
 const (
-	defaultConfigFile             = "config.json"
-	defaultSyncMaxMemoryPercent   = 25
-	defaultSyncBatchWindowSize    = 64
-	defaultTxPoolMaxMemoryPercent = 0.4
-	defaultTxPoolMaxMemorySize    = 32
+	defaultConfigFile                 = "config.json"
+	estimatedHeaderSize               = 418
```

Comment:
What is magic number of 418?
Suggest use named const variate or `unsafe.Sizeof(blk.Header)`

Reply:
It's measured from current db. It cannot be get from size because size of bytes cannot be determined from structure definition (can be any length). When node first starts without ledger, he only has genesis block, which has different header size.

Reply:
So, we should use a named const to replace it for code readability.

Reply:
This constant is defined for this purpose